recipes-domu/domu-image-android/domu-image-android.bbappend

```diff
@@ -1,28 +1,24 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
-FILESEXTRAPATHS_prepend := "${THISDIR}/../../inc:"
 
-# we need MACHINEOVERRIDES from DomD build
-do_configure[depends] += "domd-image-weston:do_domd_install_machine_overrides"
+# Most functionality and settings are defiend in xt-images.
+# Only some minimal tuning is required by environment variables.
 
-SRC_URI = " \
-    repo://github.com/xen-troops/manifests;protocol=https;branch=master;manifest=prod_devel/domu_android_host_tools.xml;scmdata=keep \
-"
+# In most cases we can use default source of android's repo, provided in xt-images
+#SRC_URI = " \
+#    repo://... \
+#"
 
-XT_BB_LAYERS_FILE = "meta-xt-prod-extra/doc/bblayers.conf.domu-image-android"
-XT_BB_LOCAL_CONF_FILE = "meta-xt-prod-extra/doc/local.conf.domu-image-android"
+# For proper build of graphics we need to set proper SOC. This info is available
+# for us from local.conf and we do not need to run do_domd_install_machine_overrides.
+# H3 ES3
+SOC_FAMILY_r8a7795 = "r8a7795"
+SOC_FAMILY_r8a7795-es3 = "r8a7795"
+# M3
+#SOC_FAMILY_r8a7796 = "r8a7796"
```

Comment:
why SOC_FAMILY_r8a7796 = "r8a7796" is commented ?

Reply:
Disabled code is removed now. Thanks for pointing.